config/zones/IN-EA.yaml

```diff
@@ -0,0 +1,20 @@
+bounding_box:
+  - - 67.434082
+    - 11.393879
+  - - 100.327148
+    - 27.664069
+capacity:
+  battery storage: 0
```

Comment:
Have we confirmed 0 storage capacity? If not it's better to leave it as Null or ommit it in my oppioion?

Reply:
good point